### ISSUE TITLE
Bash not found / composer not found running docker compose

### ISSUE THREAD

**Issue by stardothosting** [aspirepress/AspireCloud#8]
```
$ sudo docker compose run --rm webapp bash -c "composer install"
[+] Creating 1/0
 ✔ Container database  Created                                                                                                                                                      0.0s 
[+] Running 1/1
 ✔ Container database  Started                                                                                                                                                      0.2s 
/usr/local/bin/docker-php-entrypoint: exec: line 9: bash: not found
```

Then switching over to sh instead of bash : 

```
$ sudo docker compose run --rm webapp sh -c "composer install"
[+] Creating 1/0
 ✔ Container database  Created                                                                                                                                                      0.0s 
[+] Running 1/1
 ✔ Container database  Started                                                                                                                                                      0.2s 
sh: composer: not found
```

**Comment by sarah-savage** [comment]
I'm confused why this would be creating a container called `database` but if you're running `docker compose run` in our project, you should see composer run for both commands.

```
$ docker compose run --rm webapp bash -c "composer install"
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Nothing to install, update or remove
Generating autoload files
composer/package-versions-deprecated: Generating version class...
composer/package-versions-deprecated: ...done generating version class
96 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

$ docker compose run --rm webapp sh -c "composer install"
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Nothing to install, update or remove
Generating autoload files
composer/package-versions-deprecated: Generating version class...
composer/package-versions-deprecated: ...done generating version class
96 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
```

**Comment by chuckadams** [comment]
This was fixed by #12 by the way.  i should have thought to search issues when i opened the PR.
